app/helpers/routes_into_teaching_helper.rb

```diff
@@ -5,16 +5,16 @@ def answers
 
   def undergraduate_degree_summary
     {
-      "Yes" => "have a degree",
-      "No" => "do not have a degree",
-      "Not yet" => "are studying for a degree",
+      "Yes" => "have a bachelor's degree",
```

Comment:
Not sure I understand this change (and lines 9/10) - this is not something that appears on the front end?

Reply:
Ignore this - this is on the completion page!